### ISSUE TITLE
ckeditor pastebase64 pasting the image 2 times

### ISSUE THREAD

**Issue by shrachar** [ahocquet/ckeditor-pastebase64#2]
Please help us to resolve this issue

**Comment by shrachar** [comment]
Any update on this request. Please keep us posted it is urgnt.

**Comment by ahocquet** [comment]
Please describe the steps to reproduce the issue. Also details which version of ckeditor you use.

**Comment by roosephu** [comment]
I have the same problem now, with CKEditor v4.5.10.

Steps to reproduce:
1. download official CKEditor (standard) and unpack. 
2. put this plugin into `./ckeditor/plugins`. Thus we have `./ckeditor/plugins/pastebase64/plugin.js`. 
3. edit `./ckeditor/config.js`  and add an extra line of code `config.extraPlugins = 'pastebase64';`
4. open `./ckeditor/samples/index.html` and try to paste any image from any website. 
5. I see two identical images. One is specified by URL and the other is encoded by base64.

**Comment by derekm228** [comment]
Using the same version...

For #5 above, in Windows Chrome the picture specified by the URL shows the picture. In Mac Chrome the picture specified by the URL shows the URL, not the picture.

Windows pasting from Google
![image](https://cloud.githubusercontent.com/assets/2760727/17743066/79cdff10-6470-11e6-853a-c7d0d53aa474.png)

Mac pasting from Google
![image](https://cloud.githubusercontent.com/assets/2760727/17743107/9940d188-6470-11e6-9e5a-9e6c4939f6ef.png)

**Comment by derekm228** [comment]
I'm only able to reproduce this in Windows Chrome. Mac Chrome pastes the base64 image properly. The URL image mentioned in step 5 above is omitted properly in Mac Chrome.

However... as just an edge case that may be related (if it isn't then my apologies):

1. Paste an animated GIF in Windows Chrome

- The animated GIF is displayed and animates properly along with the base64 un-animating image. 

![image](https://cloud.githubusercontent.com/assets/2760727/17743428/b4e9cba0-6471-11e6-89c6-db3854007c58.png)

2. Paste an animated GIF in Mac Chrome 

- The animated GIF link is displayed along with the base 64 un-animated image

![image](https://cloud.githubusercontent.com/assets/2760727/17743479/db357be2-6471-11e6-8ea9-7e6b9ee55d54.png)

Basically my conclusion is that on a Mac, the editor still attempts to paste the image specified by URL as well, though the filtering/removal doesn't complete when it's a GIF.
